client/src/data/languages/de.js

```diff
@@ -186,7 +186,7 @@ export default {
     'HIMSELF': 'selbst',
     'ADD_USER': 'Benutzer hinzufügen',
     'UPLOAD_FILE': 'Datei hochladen',
-    'PRIVATE': 'Privatgelände',
+    'PRIVATE': 'Interne Notiz',
```

Comment:
Not sure about this change. Keep in mind we use it too for private articles

Reply:
thats not a problem.. "interne notiz" means "internal note".. 
the german word "Privatgelände" means "private property"(in terms of your house/garden or something haha) so "interne notiz" suits well